backend/sheets/sheets.py

```diff
@@ -23,24 +23,48 @@
 class Sheets(commands.Cog):
     def __init__(self, bot):
         self.bot: Client = bot
+        self.current_day = None
+        self.ink_month = 10 # October
         self.channel_description.start()
 
     def cog_unload(self):
         self.channel_description.cancel()
 
-    @tasks.loop(hours=1)
+
+    @tasks.loop(seconds=1)
     async def channel_description(self):
-        now_day = int(datetime.datetime.now().strftime("%d"))
-        channel: discord.TextChannel = self.bot.get_channel(
-            backend.config.inktober_submit_channel
-        )
-        await channel.edit(
-            reason="Time passed",
-            topic=f"Currently accepting "
-            f"{now_day - 1}: {backend.day_themes.day_themes[now_day - 1]},"
-            f"{now_day}: {backend.day_themes.day_themes[now_day]},"
-            f"{now_day + 1}: {backend.day_themes.day_themes[now_day + 1]}",
-        )
+        now_date = datetime.datetime.now(tz=datetime.timezone.utc)
+        now_day = int(now_date.strftime("%d"))
+        tomorrow_date, until_tomorrow = await self.get_tomorrow_day()
+        if now_day == self.current_day:
+            # Annoying solution to fill the feature gap of scheduled tasks not being implemented yet in d.py
+            await asyncio.sleep(until_tomorrow)
```

Comment:
Contrary to your comment this bot has recently been [upgraded to run discord.py 1.3](https://github.com/BlobEmoji/Inktober/commit/27880f3d19f6a729873ec74293d79ee930a056dd#diff-bcda051e79368cec1cf5bcaa03e35972R1) which allows you to simply pass a [`time` argument](https://discordpy.readthedocs.io/en/latest/ext/tasks/index.html#discord.ext.tasks.loop) to the decorator to have it run at a specific hour each day, avoiding all this extra code and the `get_tomorrow_day` method below.

Reply:
Can change. The library has not made a release with that feature however, which is why I designed it this way